setup.cfg

```diff
@@ -98,7 +98,12 @@ filterwarnings =
 
 [mypy]
 plugins = mypy_django_plugin.main
-disallow_untyped_calls = true
+show_error_codes = true
+strict = true
+warn_unreachable = true
```

Comment:
activate strict mode, plus error codes (useful for [type ignore comments](https://adamj.eu/tech/2021/05/25/python-type-hints-specific-type-ignore/)) and [unreachable code detection](https://adamj.eu/tech/2021/05/19/python-type-hints-mypy-unreachable-code-detection/)